### PR TITLE
Fix registry resolution for simple name packages

### PR DESCRIPTION
**Pull request by kusalk**
Follow-up PR based on [this comment](https://github.com/yoavain/fix-lockfile-integrity/pull/170/files?short_path=b335630#r1103919751).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "fix-lockfile-integrity",
-    "version": "1.3.1",
+    "version": "1.3.2",
```
Comment (yoavain) [inline package.json:3]: No need for that. The publish process promotes the version number

Comment (kusalk) [reply package.json:3]: Gotcha, will revert that :)

File: test/fixLockfileIntegrity.test.ts [hunk 2]
```diff
@@ -41,6 +41,18 @@ const LOCKFILE_V2_SCOPED_PACKAGE = {
...
 
+const SIMPLE_NAME_PACKAGE_RESOLVED = "https://registry.company.com/private/i/-/i-1.0.0.tgz";
+
+describe("Test registry parsing from resolved field", () => {
+    it("Test simple package name", () => {
+        expect(parseRegistry(LOCKFILE_V1_SIMPLE_PACKAGE.packageName.resolved, "packageName")).toEqual("https://registry.npmjs.org/");
+    });
+
+    it("Test short package name", () => {
+        expect(parseRegistry(SIMPLE_NAME_PACKAGE_RESOLVED, "i")).toEqual("https://registry.company.com/private/");
```
Comment (kusalk) [inline test/fixLockfileIntegrity.test.ts:52]: Previously was returning `https://reg`

**Review by yoavain — APPROVED** [review]
LGTM

**Pull request merged**